docs/src/configuration/routes/_index.md

```diff
@@ -1,348 +1,337 @@
 ---
-title: "Routes"
+title: Define routes
 weight: 2
-description: |
-  Platform.sh allows you to define the routes used in your environments.
-sidebarTitle: "Routes (routes.yaml)"
+description: Set things up so requests to your app are sent to the right locations.
 layout: single
 ---
 
-{{< description >}}
+You might need to control how people access your web applications,
+for example when you have [multiple apps](../app/multi-app.md) in one project.
+Or you might just want to direct requests to specific places, such as removing the `www` at the start of all requests.
 
-A route describes how an incoming HTTP request is going to be processed by Platform.sh.
-The routes are defined using the `.platform/routes.yaml` file in your Git repository.
+Control where external requests are directed by defining routes in a `.platform/routes.yaml` file in your Git repository.
 
-If you don't have one, use the commands below to create it:
+![Routes](/images/config-diagrams/routes-basic.png "0.5")
 
-```bash
-$ mkdir .platform
-$ touch .platform/routes.yaml
-```
+## Examples
 
-![Routes](/images/config-diagrams/routes-basic.png "0.5")
+These examples show how to define routes.
 
-## Route templates
+### Basic route definition
```

Comment:
I don't think it should be fully aligned with the heading below it. This one should introduce the general idea of templates. The syntax should be the same for single or multiple apps. If it's said to be "single-app", then it might get skipped when it should be the base to understand the templates. The "multi-app" one should be skipped in most cases, but is there for specific cases.

Does that make sense?

Reply:
Yeah - would it make sense to replace it with minimal instead? Basic is sometimes a negative qualifier?

Reply:
We discussed internally and agree that basic out weights its negative connotations in this context